data/mappings/info_rules.json

```diff
@@ -10,6 +10,7 @@
     # deprecated: Default `false`. Set to `true` to turn on warning when a value exists
     # invalid: Default `false`. Set to `true` to generate errors when a value exists
     # replace_with: use with a key marked deprecated or invalid to designate a replacement
+    "AUTOCORRECT_ENABLE": {"info_key": "autocorrect.enabled", "value_type": "bool"},
```

Comment:
Won't be needed if the `"features"` variant is used.

Reply:
Ah, still not familiar enough with the json/dd stuff.